### ISSUE TITLE
Avc and accessibility problems when launching Wonderous app

### ISSUE THREAD

**Issue by DiegoGW** [flutter/flutter#135877]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

1. Get latest version of Wonderous app (This is the log when this issue was created - commit a04d8bbb71c2a1a00e393218de6e44dc2ad0559a (HEAD -> main, origin/main, origin/HEAD)
Merge: 8a29d67 509487f
Author: Shawn <shawn@gskinner.com>
Date:   Mon Oct 2 14:02:12 2023 -0600
    Merge pull request #124 from yangsfang/yangsfang-patch-1
    Fix throttler calling an action twice
)
2. Run Wonderous in Android Studio (target: Pixel 6 + Android 13)
3. Navigate to the Great Wall (Navigate to: /wonder/greatWall?t=0)
4. Check the log:
```
Launching lib/main.dart on Pixel 6 in debug mode...
Running Gradle task 'assembleDebug'...
✓  Built build/app/outputs/flutter-apk/app-debug.apk.
Installing build/app/outputs/flutter-apk/app-debug.apk...
I/flutter.wonders(12265): Late-enabling -Xcheck:jni
I/flutter.wonders(12265): Using CollectorTypeCC GC.
W/flutter.wonders(12265): type=1400 audit(0.0:110): avc: denied { read } for name="max_map_count" dev="proc" ino=157967 scontext=u:r:untrusted_app:s0:c21,c257,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=com.gskinner.flutter.wonders
W/flutter.wonders(12265): Accessing hidden method Landroid/view/accessibility/AccessibilityNodeInfo;->getSourceNodeId()J (unsupported,test-api, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden method Landroid/view/accessibility/AccessibilityRecord;->getSourceNodeId()J (unsupported, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden field Landroid/view/accessibility/AccessibilityNodeInfo;->mChildNodeIds:Landroid/util/LongArray; (unsupported, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden method Landroid/util/LongArray;->get(I)J (unsupported, reflection, allowed)
Debug service listening on ws://127.0.0.1:52759/iSWKxIxN8PA=/ws
Syncing files to device Pixel 6...
```

### Expected results

None warnings should be logged

### Actual results

After launching Wonderous the following avc and accessibility warnings are logged:
```
Launching lib/main.dart on Pixel 6 in debug mode...
Running Gradle task 'assembleDebug'...
✓  Built build/app/outputs/flutter-apk/app-debug.apk.
Installing build/app/outputs/flutter-apk/app-debug.apk...
I/flutter.wonders(12265): Late-enabling -Xcheck:jni
I/flutter.wonders(12265): Using CollectorTypeCC GC.
W/flutter.wonders(12265): type=1400 audit(0.0:110): avc: denied { read } for name="max_map_count" dev="proc" ino=157967 scontext=u:r:untrusted_app:s0:c21,c257,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=com.gskinner.flutter.wonders
W/flutter.wonders(12265): Accessing hidden method Landroid/view/accessibility/AccessibilityNodeInfo;->getSourceNodeId()J (unsupported,test-api, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden method Landroid/view/accessibility/AccessibilityRecord;->getSourceNodeId()J (unsupported, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden field Landroid/view/accessibility/AccessibilityNodeInfo;->mChildNodeIds:Landroid/util/LongArray; (unsupported, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden method Landroid/util/LongArray;->get(I)J (unsupported, reflection, allowed)
Debug service listening on ws://127.0.0.1:52759/iSWKxIxN8PA=/ws
Syncing files to device Pixel 6...
```

### Code sample

<details><summary>Code sample</summary>

```Wonderous
git clone https://github.com/gskinnerTeam/flutter-wonderous-app.git
```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

<img width="1436" alt="Screenshot 2023-10-02 at 6 46 27 PM" src="https://github.com/flutter/flutter/assets/11681325/f2fee808-a2bc-4ae4-a782-903bd8e6e696">

</details>

### Logs

<details><summary>Logs</summary>

```console
Launching lib/main.dart on Pixel 6 in debug mode...
Running Gradle task 'assembleDebug'...
✓  Built build/app/outputs/flutter-apk/app-debug.apk.
Installing build/app/outputs/flutter-apk/app-debug.apk...
I/flutter.wonders(12265): Late-enabling -Xcheck:jni
I/flutter.wonders(12265): Using CollectorTypeCC GC.
W/flutter.wonders(12265): type=1400 audit(0.0:110): avc: denied { read } for name="max_map_count" dev="proc" ino=157967 scontext=u:r:untrusted_app:s0:c21,c257,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=com.gskinner.flutter.wonders
W/flutter.wonders(12265): Accessing hidden method Landroid/view/accessibility/AccessibilityNodeInfo;->getSourceNodeId()J (unsupported,test-api, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden method Landroid/view/accessibility/AccessibilityRecord;->getSourceNodeId()J (unsupported, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden field Landroid/view/accessibility/AccessibilityNodeInfo;->mChildNodeIds:Landroid/util/LongArray; (unsupported, reflection, allowed)
W/flutter.wonders(12265): Accessing hidden method Landroid/util/LongArray;->get(I)J (unsupported, reflection, allowed)
Debug service listening on ws://127.0.0.1:52759/iSWKxIxN8PA=/ws
Syncing files to device Pixel 6...
```

</details>

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
[✓] Flutter (Channel master, 3.16.0-2.0.pre.8, on macOS 13.5 22G74 darwin-arm64, locale en-US)
    • Flutter version 3.16.0-2.0.pre.8 on channel master at /Users/diegogawenda/fvm/versions/master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b71f8ce7ae (2 hours ago), 2023-10-02 15:09:38 -0400
    • Engine revision f3a92cb0c4
    • Dart version 3.2.0 (build 3.2.0-221.0.dev)
    • DevTools version 2.28.1

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/diegogawenda/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/diegogawenda/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/diegogawenda/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15A240d
    • CocoaPods version 1.12.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)

[✓] VS Code (version 1.80.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.74.0

[✓] Connected device (4 available)
    • Pixel 6 (mobile)         • 1A141FDF600823            • android-arm64  • Android 13 (API 33)
    • Somnio’s iPhone (mobile) • 00008120-001168D83C87C01E • ios            • iOS 17.0.2 21A351
    • macOS (desktop)          • macos                     • darwin-arm64   • macOS 13.5 22G74 darwin-arm64
    • Chrome (web)             • chrome                    • web-javascript • Google Chrome 117.0.5938.92

[✓] Network resources
    • All expected network resources are available.
```

</details>

**Comment by DiegoGW** [comment]
Duplicates https://github.com/flutter/flutter/issues/135876
